src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,47 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+    private final List<Car> cars;
+    private int cycle;
+
+    public Cars() {
+        this.cars = new ArrayList<>();
+    }
+
+    public void addCars(String carName) {
+        cars.add(new Car(carName));
+    }
+
+    public List<Car> selectWinners() {
+        int winnerScore = selectWinnerScore(this.cars);
+
+        return this.cars.stream()
+                .filter(cars -> cars.isWinner(winnerScore))
+                .map(cars -> new Car(cars.getName()))
+                .collect(Collectors.toList());
+    }
+
+    private int selectWinnerScore(List<Car> carInfo) {
+        return carInfo.stream()
+                .max(Comparator.comparingInt(Car::getLocation))
+                .get()
+                .getLocation();
```

Comment:
다루는 값이 기본형(primitive type)인 경우, 기본형 스트림(= IntStream, DoubleStream, LongStream...)을 이용해서 동일한 로직을 아래와 같이 변경해볼 수도 있을 것 같아요 😃 (위의 코멘트처럼 바로 get을 하는 것 보다 적절한 처리를 해야겠죠? 😉)
```suggestion
        return carInfo.stream()
                .mapToInt(Car::getLocation)
                .max()
                .getAsInt();
```